gulpfile.js

```diff
@@ -31,6 +31,9 @@ var paths = {
 		extensions: ['Extensions/**/*.css'],
 		themes: ['Themes/**/*.css']
 	},
+	paperboy: [
+		'Paperboy/*'
```

Comment:
I think this is still an outstanding bug [per discussion on Gitter](https://gitter.im/new-xkit/XKit?at=56b75214a44cf16b6de054b3)?  See [my response](https://gitter.im/new-xkit/XKit?at=56b773772bb6eb8230a1f5d8).